workflows/alevin-quant/run-alevin.nf

```diff
@@ -48,12 +49,14 @@ process alevin{
 }
 
 workflow{
-  sample_ids = params.sample_ids?.tokenize(',') ?: []
-  ch_reads = Channel.fromList(sample_ids)
+  run_ids = params.run_ids?.tokenize(',') ?: []
+  ch_reads = Channel.fromPath(params.run_metafile)
+    .splitCsv(header: true, sep: '\t')
+    .filter{it.scpca_run_id in run_ids} // use only the rows in the sample list
     // create tuple of [sample_id, [Read1 files], [Read2 files]]
-    .map{ id -> tuple("$id",
-                      file("${params.sample_dir}/${id}/*_R1_*.fastq.gz"),
-                      file("${params.sample_dir}/${id}/*_R2_*.fastq.gz"),
+    .map{row -> tuple(row.scpca_run_id,
```

Comment:
If so, does this mean that this is also acting as `i`, allowing iteration on rows in the metadata file.

Reply:
Yep! I could have used `it` here too, but row makes more sense?